frontend/src/component/common/Table/cells/FeatureSeenCell/LastSeenTooltip.tsx

```diff
@@ -73,64 +79,66 @@ export const LastSeenTooltip = ({
-                            <StyledDescriptionBlock key={name}>
-                                <StyledDescriptionBlockHeader>
-                                    {name}
-                                </StyledDescriptionBlockHeader>
-                                <StyledValueContainer>
-                                    <ConditionallyRender
-                                        condition={Boolean(lastSeenAt)}
-                                        show={
-                                            <TimeAgo
-                                                date={lastSeenAt!}
-                                                title=''
-                                                live={false}
-                                                formatter={(
-                                                    value: number,
-                                                    unit: string,
-                                                    suffix: string,
-                                                ) => {
-                                                    const [, textColor] =
-                                                        getColor(unit);
-                                                    return (
-                                                        <StyledValue
-                                                            color={textColor}
-                                                        >
-                                                            {`${value} ${unit}${
-                                                                value !== 1
-                                                                    ? 's'
-                                                                    : ''
-                                                            } ${suffix}`}
-                                                        </StyledValue>
-                                                    );
-                                                }}
-                                            />
-                                        }
-                                        elseShow={
-                                            <StyledValue
-                                                color={defaultTextColor}
-                                            >
-                                                no usage
-                                            </StyledValue>
-                                        }
-                                    />
-                                </StyledValueContainer>
-                            </StyledDescriptionBlock>
-                        ))}
-                    </>
+                    <StyledListContainer>
+                        {environments?.map(({ name, lastSeenAt, yes, no }) => {
+                            return (
+                                <StyledDescriptionBlock key={name}>
```

Comment:
this is already too deep. I suggest extract refactoring

Reply:
This was mistake, actually this big change is not needed. Removed.